Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -38,14 +38,22 @@ button.webButton.primary
   border: none;
 }
 
-button.webButton.primary:hover,
-button.webButton.primary:focus
+button.webButton.primary:not(:disabled):hover,
+button.webButton.primary:not(:disabled):focus
 {
   background-color: var(--color-primary-highlight);
 }
 
-button.webButton.secondary
+button.webButton.supplemental
 {
+  margin: var(--remotion-themed-spacing);
```

Comment:
why do we need margin here. can we make this to work just like the other button types, only without border?

Reply:
Isn't the supplemental button supposed to be smaller than the other buttons?

Reply:
No, it's just borderless. The hover highlight etc may still use the full size of the button.